cmd/tendermint/commands/reset_priv_validator.go

```diff
@@ -31,6 +32,22 @@ func MakeResetAllCommand(conf *config.Config, logger log.Logger) *cobra.Command
 	return cmd
 }
 
+// MakeResetAllCommand constructs a command that removes the database of
+// the specified Tendermint core instance.
+func MakeResetStateCommand(conf *config.Config, logger log.Logger) *cobra.Command {
+	var keyType string
+
+	cmd := &cobra.Command{
+		Use:   "unsafe-reset-state",
```

Comment:
should we leave the "unsafe as a different operation?"

Reply:
changed to reset-state as its not entirely unsafe anymore